### PR TITLE
[12.0] Added migrated code from v11 to v12.

### PR DESCRIPTION
**Pull request by nikeshscs**
Hello sir,
here i send pull request of fleet_management modules after migrating from v11 to v12. 
Thank you.

### CODE REVIEW

File: fleet_operations/models/fleet_extended.py [hunk 1]
```diff
@@ -273,7 +271,7 @@ def _compute_vehicle_name(self):
     description = fields.Text(string='About Vehicle', translate=True)
     engine_size = fields.Char(string='Engine Size', size=16)
     cylinders = fields.Integer(string='No of Cylinders')
-    front_tire_size = fields.Float(string='Size')
+    front_tire_size = fields.Float(string='Size ')
```
Comment (JayVora-SerpentCS) [inline fleet_operations/models/fleet_extended.py]: no extra space

File: fleet_operations/models/fleet_extended.py [hunk 2]
```diff
@@ -282,12 +280,12 @@ def _compute_vehicle_name(self):
     last_odometer = fields.Float(string='Last Service Odometer')
     last_odometer_unit = fields.Selection([('kilometers', 'Kilometers'),
                                            ('miles', 'Miles')],
-                                          string='Odometer Unit',
+                                          string='Odometer Unit ',
```
Comment (JayVora-SerpentCS) [inline fleet_operations/models/fleet_extended.py]: no extra space

File: fleet_operations/models/fleet_extended.py [hunk 3]
```diff
@@ -317,12 +315,12 @@ def _compute_vehicle_name(self):
                                        string='Attachments')
     sale_purchase_attach_ids = fields.One2many('ir.attachment',
                                                'attachment_id_2',
-                                               string='Attachments')
+                                               string='Attachments ')
```
Comment (JayVora-SerpentCS) [inline fleet_operations/models/fleet_extended.py]: no extra space

File: fleet_operations/models/fleet_extended.py [hunk 4]
```diff
@@ -317,12 +315,12 @@ def _compute_vehicle_name(self):
...
     odometer = fields.Float(compute='_get_odometer', inverse='_set_odometer',
                             string='Last Odometer',
                             help='Odometer measure of the vehicle at the \
                                 moment of this log')
-    vehical_color_id = fields.Many2one('color.color', string='Color')
+    vehical_color_id = fields.Many2one('color.color', string='Color ')
```
Comment (JayVora-SerpentCS) [inline fleet_operations/models/fleet_extended.py]: no extra space

File: fleet_operations/models/fleet_extended.py [hunk 5]
```diff
@@ -836,7 +848,7 @@ class FleetWittenOff(models.Model):
     engine_repair = fields.Boolean(string='ENGINE REPAIR')
     electrical_system = fields.Boolean(string='ELECTRICAL SYSTEM')
     exhaust_system = fields.Boolean(string='EXHAUST SYSTEM')
-    location_id = fields.Many2one('vehicle.location', string='Location')
+    location_id = fields.Many2one('vehicle.location', string=' Location ')
```
Comment (JayVora-SerpentCS) [inline fleet_operations/models/fleet_extended.py]: no extra space

File: fleet_operations/models/fleet_service.py [hunk 6]
```diff
@@ -29,6 +30,8 @@ def unlink(self):
 
 
 class FleetVehicleLogServices(models.Model):
+    _inherit = 'fleet.vehicle.log.services'
```
Comment (JayVora-SerpentCS) [inline fleet_operations/models/fleet_service.py:33]: def copy(self, default=None):
         if not default:
             default = {}
         raise Warning(_('You can\'t duplicate record!'))
return super(ServiceCategory, self).copy(default=default)

>>> raise pa6i return?? na jaamyu

File: oca_dependencies.txt [hunk 7]
```diff
@@ -1 +1,3 @@
-reporting-engine
\ No newline at end of file
+reporting-engine
+ow_account_asset
+ow_account_budget
```
Comment (JayVora-SerpentCS) [inline oca_dependencies.txt:3]: mention the module names in depends manifest too

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by nikeshscs** [comment]
Hello @JayVora-SerpentCS,
We improved code as per requested changes.
Thank you.

**Comment by JayVora-SerpentCS** [comment]
@YogendrasinhChavda please review

File: fleet_operations/models/fleet_extended.py (hunk 1 above)
Comment (YogendrasinhChavda) [reply fleet_operations/models/fleet_extended.py]: @JayVora-SerpentCS Done.

**Comment by YogendrasinhChavda** [comment]
@JayVora-SerpentCS 
Changes are reviewed.

**Pull request merged**